tools/development/helpers/install-python.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Physics
+# @file           tools/development/helpers/install-python.sh
+# @author         Lucas Brémond <lucas@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+PYTHON_VERSION="3.9"
+
+################################################################################################################################################################
+
+project_directory="$(git rev-parse --show-toplevel)"
+python_directory="${project_directory}/build/bindings/python/OpenSpaceToolkitPhysicsPy-python-package-${PYTHON_VERSION}"
+
+pushd "${python_directory}" > /dev/null
+
+pip install .
+
+popd > /dev/null
+
```

Comment:
Those two `.sh` files are great. Let's batch apply to the other repositories, I'll check with Antoine :)

Reply:
`--force-reinstall` useful when cross-libraries building and development is happening